src/scipp/plotting/controller1d.py

```diff
@@ -33,6 +33,9 @@ def rescale_to_data(self, button=None):
         with_min_padding = self.vmin is None
         with_max_padding = self.vmax is None
         vmin, vmax = self.find_vmin_vmax(button=button)
+        if vmin.unit is None:
+            vmin.unit = ''
+            vmax.unit = ''
```

Comment:
```suggestion
            vmin.unit = 'one'
            vmax.unit = 'one'
```

In my opinion, we should make this a (informal) style guide. I find it confusing to use both empty strings and `None` even though both mean different things.